polaris-icons/icons/PlanMajor.yml

```diff
@@ -0,0 +1,16 @@
+name: Plans
+set: major
+description: Wayfinding | Used to help direct merchants to the Plan page in Settings.
```

Comment:
The description should probably just be a sentence without the ` | ` divider.

If we need more categorization or better search, that is better done through front matter so that we can parse the data in a consistent matter.